### ISSUE TITLE
Crear filtro para eliminar pruebas

### ISSUE THREAD

**Issue by JJ** [proyectopetra/grafo#2]
Como veis aquí https://twitter.com/jjmerelo/status/609313580857577472/photo/1 hay enlaces BT que no son muy lógicos, que pueden deberse a las pruebas iniciales, En general, todos los UGR -> Sevilla tienen entre 1 y 10 pasos. ¿Elimino todas las aristas que tengan menos de 10 como no significativas? También @pacastillo y @fergunet pueden contestar.

**Comment by pacastillo** [comment]
Sí, algunos registros pueden ser de pruebas que hicieran en Ciudad2020.
Vamos a probar a ver qué sale al eliminarlas.
